.github/ISSUE_TEMPLATE/default_issue.yml

```diff
@@ -0,0 +1,61 @@
+name: New Issue
+title: Neon Issue
+description: Use this template for all issues that are not bugs.
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks in advance for taking the time to file this issue! 
```

Comment:
Let's make the structure for this kind of issue report extremely minimal, so that we minimize the risk of people giving up on filing issues.

Reply:
I think this is a good idea. Make the "other" issue nothing but an open text field.